client/verta/verta/_deployment/endpoint.py

```diff
@@ -227,6 +237,10 @@ def update_from_config(self, filepath):
         return self._update_from_dict(update_dict)
 
     def _update_from_dict(self, update_dict):
+        if (not "model_version_id" in update_dict and not "run_id" in update_dict) or \
+                ("model_version_id" in update_dict and "run_id" in update_dict):
```

Comment:
Line breaks `\` are discouraged in Python; using parentheses are preferred
```suggestion
        if ((not "model_version_id" in update_dict and not "run_id" in update_dict)
                or ("model_version_id" in update_dict and "run_id" in update_dict)):
```

Reply:
Resolved below.